python/main.py

```diff
@@ -1,7 +1,7 @@
 import inspect
 import json
 
-from sidecar import chat, cli, ingest, rewrite, storage, search
+from sidecar import chat, cli, ingest, rewrite, search, storage
```

Comment:
Not necessary for this PR, but is there a linter/formatter that's not running as part of CI?

Reply:
Honestly, this is something my vs-code only does for refstudio and I haven't been able to figure out why

Reply:
I had thought it might have been due to prettier integration, but seems not